specification/compute/resource-manager/Microsoft.Compute/stable/2021-04-01/examples/CreateOrUpdateAScaleSetWithCapacityReservation.json

```diff
@@ -0,0 +1,230 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscription-id}",
+    "resourceGroupName": "myResourceGroup",
+    "vmScaleSetName": "{vmss-name}",
+    "api-version": "2021-07-01",
+    "parameters": {
+      "sku": {
+        "tier": "Standard",
```

Comment:
>"tier": "Standard", [](http://example.com/codeflow?start=8&length=19)

is this needed?

Reply:
this is just what i use when i test using arm template thru portal